projects/sandbox/export/export/main.py

```diff
@@ -84,11 +83,10 @@ def main(
             during whitening
         highpass:
             Frequency to use for a highpass filter
-        weights:
+        weights_dir:
```

Comment:
I think you should still be able to specify a full path to weights named whatever you like. Maybe do a check and if `weights.is_dir()` then `weights = weights / "weights.pt"`

Reply:
Agreed. I rearranged things a little, let me know what you think.